src/components/side-navigation/NavigationHeader.module.sass

```
@@ -0,0 +1,2 @@
+.navigation-header
+    padding-block: 0.75em
```

`padding` shouldn't be applied on top.
```suggestion
.navigation-header
    padding-block-end: 0.75em
```

---

Agree to disagree